ibis/impala/client.py

```diff
@@ -295,10 +345,13 @@ def _db_type_to_dtype(self, db_type):
 
 def _column_batches_to_dataframe(names, batches):
     from ibis.compat import zip as czip
-    cols = {}
-    for name, chunks in czip(names, czip(*[b.columns for b in batches])):
-        cols[name] = _chunks_to_pandas_array(chunks)
-    return pd.DataFrame(cols, columns=names)
+    try: #Give Space for turbodbc type
```

Comment:
It looks like you have quite a few lint errors: https://circleci.com/gh/pandas-dev/ibis/128?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

I've found it helpful to integrate `flake8` with my editor so that every time I save, `flake8` will run and report any errors in the editor. Then, I can fix them up right away. Most common editors (vim, emacs, and pycharm) support this kind of integration.